### DISCUSSION TITLE
Scraping template

### DISCUSSION THREAD

**Discussion by djlauria** [FreshRSS/FreshRSS#6457] [Q&A] [answered]
Obviously, this can be done with two open windows and a bunch of copy/paste, but: is there a way to create a template to use for scraping? Let's say for example that you follow several writers at an online magazine. If anything changes with the web site, each of them has to be updated separately, even though their XPath information is identical. A template for that site would also allow you to add another writer quickly and easily. 
Apologies if this exists and my searching hasn't turned it up yet.

**Comment by math-GH** [comment, marked as answer]
With the upcoming 1.24.0 it will be possible to export the settings of a feed as OPML file, including the scraping information

**Reply by Alkarex** [reply]
Already available in our rolling-release `edge` version, and tests + feedback welcome!
